src/toasts/toasts.js

```diff
@@ -1,24 +1,28 @@
 import React from "react";
 import ReactDOM from "react-dom";
+import { Subject } from "rxjs";
+import ActiveToasts from "./active-toasts.component";
+
+export const toastsSubject = new Subject();
+let toastId = 0;
```

Comment:
To keep track of which toast is toast, we assign the toasts an id that we can use as a React `key`.